pkg/acceptance/testing.go

```diff
@@ -72,12 +73,17 @@ func init() {
 		log.Panicf("Cannot instantiate new secondary client, err: %v", err)
 	}
 	atc.secondaryClient = secondaryClient
+
+	atc.testClient = helpers.NewTestClient(client, TestDatabaseName, TestSchemaName, TestWarehouseName)
+	atc.secondaryTestClient = helpers.NewTestClient(secondaryClient, TestDatabaseName, TestSchemaName, TestWarehouseName)
 }
 
 type acceptanceTestContext struct {
-	config          *gosnowflake.Config
-	client          *sdk.Client
-	secondaryClient *sdk.Client
+	config              *gosnowflake.Config
+	client              *sdk.Client
```

Comment:
are we targeting to remove those when the TestClients will be fully functional and used in every test?

Reply:
yes